app/views/records/index.html.erb

```diff
@@ -1,8 +1,22 @@
 <% if @task && @product_test %>
   <h1>Patient List</h1>
-  <%= button_to html_patients_product_test_path(@product_test), :method => :get, :class => "btn btn-default pull-right" do %>
-    <i class="fa fa-fw fa-download" aria-hidden="true"></i> Download HTML Patients
-  <% end %>
+  <div>
```

Comment:
I'd add the `pull-right` class to this div and remove it from the button and dropdown (and you'll have to switch the button and dropdown so they appear in the correct order). Otherwise looks great!